src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -384,16 +386,100 @@
 (rf/reg-event-fx
  :wallet/set-token-amount-to-send
  (fn [{:keys [db]} [{:keys [amount stack-id start-flow?]}]]
+   {:db (assoc-in db [:wallet :ui :send :amount] amount)
+    :fx [[:dispatch
+          [:wallet/wizard-navigate-forward
+           {:current-screen stack-id
+            :start-flow?    start-flow?
+            :flow-id        :wallet-send-flow}]]]}))
+
+(rf/reg-event-fx
+ :wallet.send/auth-slider-completed
```

Comment:
When slider moved by user we initiate sequence of building transaction and signing it